src/token/api/src/canister.rs

```diff
@@ -250,13 +250,13 @@ pub trait TokenCanisterAPI: Canister + Sized {
     /// Note, that the `value` cannot be less than the `fee` amount. If the value given is too small,
     /// transaction will fail with `TxError::AmountTooSmall` error.
     #[cfg_attr(feature = "transfer", update(trait = true))]
-    fn icrc1_transferIncludeFee(
+    fn transferIncludeFee(
```

Comment:
You should also change the `icrc1_transfer_include_fee` to `transfer_include_fee`.